hail/python/test/hail/test_ir.py

```diff
@@ -101,13 +101,13 @@ def value_irs(self):
             ir.MatrixToValueApply(matrix_read, {'name': 'ForceCountMatrixTable'}),
             ir.TableAggregate(table, ir.MakeStruct([('foo', ir.ApplyAggOp('Collect', [], [ir.I32(0)]))])),
             ir.TableWrite(table, ir.TableNativeWriter(new_temp_file(), False, True, "fake_codec_spec$$")),
-            ir.TableWrite(table, ir.TableTextWriter(new_temp_file(), None, True, 0, ",")),
+            ir.TableWrite(table, ir.TableTextWriter(new_temp_file(), None, True, "concatenated", ",")),
             ir.MatrixAggregate(matrix_read, ir.MakeStruct([('foo', ir.ApplyAggOp('Collect', [], [ir.I32(0)]))])),
             ir.MatrixWrite(matrix_read, ir.MatrixNativeWriter(new_temp_file(), False, False, "", None, None)),
             ir.MatrixWrite(matrix_read, ir.MatrixNativeWriter(new_temp_file(), False, False, "",
                                                               '[{"start":{"row_idx":0},"end":{"row_idx": 10},"includeStart":true,"includeEnd":false}]',
                                                               hl.dtype('array<interval<struct{row_idx:int32}>>'))),
-            ir.MatrixWrite(matrix_read, ir.MatrixVCFWriter(new_temp_file(), None, False, None)),
+            ir.MatrixWrite(matrix_read, ir.MatrixVCFWriter(new_temp_file(), None, 'concatenate', None)),
```

Comment:
Seems like you're maybe missing a "d" here? Like, should be "concatenated"? Unless I'm misreading aand this function takes a different set of strings than the above. This kind of makes me wish we had a Python equivalent of `ExportType.scala` so we could write like `ExportType.CONCATENATED` or something here and avoid spelling error bugs. Or maybe we have a way of specifying a more specific type than `str`? Like `oneof('concatened', 'parallel', ....)`?

Reply:
Excellent suggestions.  Done.  Much cleaner.